hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/action/Action.java

```diff
@@ -59,9 +47,6 @@
     @JsonInclude(Include.NON_NULL)
     private Event event;
 
-    @JsonInclude(Include.NON_EMPTY)
-    private Map<String, String> properties;
-
     @JsonInclude(Include.NON_NULL)
     private String result;
```

Comment:
Not at the moment.
I think is too much verbosity.
Today we have:
- Timestamp on the data used for generate event/alert.
- Timestamp on event creation and alert lifecycle (open, ack, resolve).
- Timestamp on action creation.
So, an actionEndTime would give more verbosity, but as the async nature of the Action, I'm not fully sure if this is needed.
We can add it, no big deal on that, but I thought in my first implementation that storing the result should be enough.
Perhaps, I can add it later as a separate tasks if needed.